spec/tree/values/PhrasingContentSpec.js

```diff
@@ -0,0 +1,32 @@
+import FormattingElement from "../../../src/tree/values/FormattingElement";
+
+describe( "PhrasingContent", () => {
+	it( "can make a new PhrasingContent object", () => {
+		const formattingElement = new FormattingElement( "strong", 5, 30, { id: "some-id" } );
+
+		expect( formattingElement.tag ).toEqual( "strong" );
+		expect( formattingElement.start ).toEqual( 5 );
+		expect( formattingElement.end ).toEqual( 30 );
+		expect( formattingElement.attributes ).toEqual( { id: "some-id" } );
+	} );
+
```

Comment:
Missing the `getAttributeString` specs. Mostly to test some unhappy paths.

Reply:
Added a test.
Most unhappy paths (ill-formed attributes) will be caught by `htmlparser5` is my guess, so I am a bit hesitant to add more.
We can always adds more tests later on.